app/components/welcome/welcome.js

```diff
@@ -156,20 +163,75 @@ const dialogs = [
       </>
     )
   },
+  ({ page, next, previous }) => {
+    return (
+      <>
+        <Back page={page} onClick={previous} />
+        <Illustration>
+          <IllustrationVault />
+        </Illustration>
+        <Heading>Introducing the Vault</Heading>
+        <p>
+          We use peer-to-peer technology: Hypergraph is powered by the people using it!
+          Share your content as you'd like, without anybody getting in between.
+        </p>
+        <p>
+          Hypergraph Vault makes your work available even when your computer is offline. Plus, we're
+          working with libraries on archiving it for future generations! 👵👨👶
```

Comment:
```suggestion
          working with libraries on archiving it for future generations! 👵🏾👴🏻👶
```

My only comment on this is for a bit more diversity in the emojis. @lhehnke what do you think? 

<img width="1045" alt="Screenshot 2020-08-04 at 18 03 26" src="https://user-images.githubusercontent.com/2946344/89316767-ce151b80-d67c-11ea-90f6-d08983b13220.png">

Otherwise looks good to me and I'll approve once this is discussed :)